pkg/virt-handler/cache/cache_test.go

```diff
@@ -314,6 +324,8 @@ var _ = Describe("Domain informer", func() {
 
 			Expect(ok).To(BeTrue())
 			Expect(val).To(Equal("some-value"))
+			Expect(eventDomain.Status.OSInfo.Name).To(Equal("fedora"))
+			Expect(eventDomain.Status.Interfaces[0].Name).To(Equal("ethx"))
```

Comment:
This test is the one that unit tests the resync, and waits for the timer of it 
i extended the resync, hence extended the test
you can see the PR that introduced the resync + the unit test of it at
https://github.com/kubevirt/kubevirt/pull/3893

This btw also why i didn't test some of the flows of non empty, because we already have them here
referring for example to your comment
https://github.com/kubevirt/kubevirt/pull/5560#discussion_r626266763
unless its need to be doubled

Btw this change was already in the original PR that was reverted.

Reply:
We check that the data is added to the resync flow is working.
I think it worth to keep it, we can add other unit tests as well if needed, but i wouldn't drop it
wdyt?

Reply:
The resync flow is working because it is tested with some arbitrary/sample data. If we need to test all data options, then we would not be able to maintain such a huge test.

Maybe I can illustrate it like this:
A is the resync mechanism.
B is the data population.

Production flow is: A ==>> B
You changed B.
A was not changed, therefore it requires no special coverage.
B was changes, it requires coverage of the new scenarios.

Testing A to validate B changes is not advised, it overloads A with indirect implications which come from B.

Reply:
ok will drop